apps/worker/src/app/workflow/usecases/add-job/add-delay-job.usecase.ts

```diff
@@ -36,11 +37,17 @@ export class AddDelayJob {
 
     let delay;
 
+    const subscriber = await this.subscriberRepository.findOne({
+      _id: data._subscriberId,
+      _environmentId: data._environmentId,
+    });
+
     try {
       delay = this.computeJobWaitDurationService.calculateDelay({
         stepMetadata: data.step.metadata,
         payload: data.payload,
         overrides: data.overrides,
+        timezone: subscriber?.timezone,
```

Comment:
Why is this needed on the delay job? I thought we only needed this in scheduled digest?

Reply:
I'll remove it